feature/Add-custom-ReturnArrayClassMethodToYieldRector (#3)
eonx-com/easy-quality: docs/ecs/sniffs.md

]
```

##### [\EonX\EasyQuality\Sniffs\Methods\UseYieldInsteadReturnSniff][14]

> Checks that using `yield` instead `return` in a specified classes and methods.

Reviewer: ```suggestion
Checks that `yield` is used instead of `return` in specified classes and methods.
```

Author: Done

**Parameters**

> - `applyTo` - An array of regular expressions to match method names. Format.

Reviewer: ```suggestion
- `applyTo` - An array of regular expressions to match the class namespace and method name. Example:
```

Author: Done
